generative/maisi/README.md

```diff
@@ -0,0 +1,84 @@
+# Medical AI for Synthetic Imaging (MAISI)
+This example demonstrates the applications of training and validating NVIDIA MAISI (Medical AI for Synthetic Imaging), a 3D Latent Diffusion Model (LDM) capable of generating large CT images accompanied by corresponding segmentation masks. It supports variable volume size and voxel spacing and allows for the precise control of organ/tumor size.
+
+## MAISI Model Highlight
+- A Foundation VAE model for latent feature compression that works for both CT and MRI with flexible volume size and voxel size
+- A Foundation Diffusion model that can generate large CT volumes up to 512 &times; 512 &times; 768 size, with flexible volume size and voxel size
+- A ControlNet to generate image/mask pairs that can improve downstream tasks, with controllable organ/tumor size
+
+## Example Results and Evaluation
```

Comment:
Shall we have placeholders for 
- training curve
- metrics 
here?

Reply:
Could we add them in the next PR? It needs some time to generate results and decide what we will add

Reply:
Sure, I'm working on it.

Reply:
Yeah, especially, for diffusion Unet and ControlNet, the loss curve cannot reflect the training progress. However, we can add some generated results.

Reply:
Sure closing it for now